src/views/GradeView/FinalGrade.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { Fragment } from 'react';
+
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import CollapsibleFeedback from 'components/CollapsibleFeedback';
+import { useAssessmentsData } from 'data/services/lms/hooks/selectors';
+import messages from './messages';
+import AssessmentCriterion from 'components/CollapsibleFeedback/AssessmentCriterion';
+
+const FinalGrade = () => {
+  const { formatMessage } = useIntl();
+  const assessments = useAssessmentsData();
+
+  const result = [];
+  let finalStepScore = null;
+  if (assessments.staff) {
+    finalStepScore = assessments.staff.stepScore;
+    const stepName = formatMessage(messages.staffStep);
+    result.push(
+      <CollapsibleFeedback
+        stepName={stepName}
+        stepScore={assessments.staff.stepScore}
+      >
+        <AssessmentCriterion {...assessments.staff.assessment} stepName={stepName} />
+      </CollapsibleFeedback>
+    );
+  }
+  if (assessments.peer) {
+    finalStepScore = finalStepScore || assessments.peer.stepScore;
+    const stepName = formatMessage(messages.peerStep);
```

Comment:
pls use `stepLabel` if you are talking about a message.  Mostly because we have a global data object called stepNames and use that as a key in many places.